addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -171,7 +171,7 @@
                                 attrs="{'readonly': [('state', '!=', 'draft')]}" force_save="1"/>
                             <label for="subject">Subject</label>
                             <div class="o_mass_mailing_subject d-flex flex-row align-items-baseline">
-                                <field class="text-break" name="subject" string="Subject"
+                                <field class="text-break o_mail_emojis_char_field" name="subject" string="Subject"
```

Comment:
why do we need to add this class ? (same below)

Reply:
We don't but I hadn't figured out widgets got an assigned css class by that point ^^

Will remove